src/groups/mqb/mqbblp/mqbblp_queueenginetester.cpp

```diff
@@ -502,6 +502,8 @@ void QueueEngineTester::init(const mqbconfm::Domain& domainConfig)
                                                 k_NULL_QUEUE_KEY,
                                                 k_PARTITION_ID,
                                                 d_mockDomain_mp.get(),
+                                                0,
+                                                // not counting Unconfirmed
```

Comment:
This comment is misleading, since we actually count unconfirmed. We just don't pass a parent counter to `QueueState`

Reply:
That meant the caller is not counting
(`// no parent counter of unconfirmed`)